scenarios/action_recognition/00_webcam.ipynb

```diff
@@ -1,5 +1,14 @@
 {
```

Comment:
<p>[very minor] maybe only show eg 10 to save some space</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
done